haystack/pipelines/utils.py

```diff
@@ -161,7 +166,10 @@ def print_eval_report(
     graph: DiGraph,
     n_wrong_examples: int = 3,
     metrics_filter: Optional[Dict[str, List[str]]] = None,
-    doc_relevance_col: str = "gold_id_or_answer_match",
```

Comment:
Tutorial 5 needs to be updated because of that change. There is a comment mentioning `doc_relevance_col`.